spec/integration/login_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
```

Comment:
- [ ] When I am running your test they are not passing, Please add more instructions on how to run your tests or fix the issue 😉

Reply:
👍🏽 Addressed ✔️